tests/e2e/normalize.go

```diff
@@ -44,6 +44,7 @@ func normalizeObject(u *unstructured.Unstructured, project testgcp.GCPProject, u
 
 	visitor.replacePaths = map[string]any{}
 	visitor.replacePaths[".metadata.deletionTimestamp"] = "1970-01-01T00:00:00Z"
+	visitor.replacePaths[".status.creationTimestamp"] = "1970-01-01T00:00:00Z"
```

Comment:
I'm not very familiar with this part of the code, but this seems to be reused by a few test flows. Have you verified that other test flows are not impacted?

Reply:
This function replaces random values with a fixed string so that the values are identical and predictable during a golden comparison. This function is only used for golden testing and would not affect anything:
- If the item to be normalized does not have `status. creationTimestamp`  field, nothing changes.
- If the item to be normalized has `status. creationTimestamp` field, it's necessary to replace it or the golden test might fail.